### ISSUE TITLE
How to call model.evaluate() about multi-task ?

### ISSUE THREAD

**Issue by FallenTransformer** [keras-team/autokeras#1574]
The dataset has 5 classification targets, I want to create a multi-task model, but there always an error when I call model.evaluate(). The follow is my code. 

import pandas as pd
import autokeras as ak
from tensorflow.keras.utils import plot_model
import tensorflow as tf

# get the dataset from data.csv file
df = pd.read_csv("data.csv") 

# split the dataset
test = df.sample(frac=0.2, random_state=123) 
train = df.drop(test.index) #train

train_target_1 = train.pop("target1")
train_target_2 = train.pop("target2")
train_target_3 = train.pop("target3")
train_target_4 = train.pop("target4")
train_target_5 = train.pop("target5")

test_target_1 = test.pop("target1")
test_target_2 = test.pop("target2")
test_target_3 = test.pop("target3")
test_target_4 = test.pop("target4")
test_target_5 = test.pop("target5")

# create a multi-task auto model with one StructuredDataInput node and 5 ClassificationHead
clf = ak.AutoModel(inputs=[ak.StructuredDataInput()],
                                outputs=[
                                ak.ClassificationHead(metrics=["accuracy"], name="head-1"),
                                ak.ClassificationHead(metrics=["accuracy"], name="head-2"),
                                ak.ClassificationHead(metrics=["accuracy"], name="head-3"),
                                ak.ClassificationHead(metrics=["accuracy"], name="head-4"),
                                ak.ClassificationHead(metrics=["accuracy"], name="head-5")],
                                overwrite=True,
                                max_trials=10)

# fit the model
clf.fit([train], [train_target_1, train_target_2, train_target_3, train_target_4, train_target_5],
        epochs=10,
        validation_split=0.25,
        verbose=True,
        shuffle=True,
        validation_freq=1,
        batch_size=256)

model = clf.export_model()
predict = model.predict(test)
print(predict)
plot_model(model, to_file="mymodel.png", show_shapes=True, show_layer_names=True, show_dtype=True)
model.evaluate([test], [test_target_1, test_target_2, test_target_3, test_target_4, test_target_5])
# when I call model.evaluate() to evaluate the model on my test set,  but I always get an error and I can't find an available solution in Autokeras API documentation, so how can I call the method correctly? Thank you!
